config.js

```diff
@@ -4,11 +4,18 @@
 //
 //
 var trace = true; // trace entry and exit for debugging
-
-
+var generate_item_list = true; //generate the item, weapon, and armor lists for the inventory tab on startup
+var skill_list_on = true; //regenerate skill list ability on change to skill tab and skill name change
+var weapon_list_on = true; //regenerate weapon list ability on change to weapontab and weapon name change
+var realtime = true; //character sheet log shows real time 
+var gametime = true; //character sheet log shows game time 
+var attack_template = "harn-fancy"; // sets the roll template (harn-fancy or harnroll) for the attack function
+var defend_template = "harn-defend"; // sets the roll template (harn-defend or harnroll) for the defend function
+var emlmax = 97; // canon 95
```

Comment:
Move these two to under House rules, and perhaps make it a single toggle, "extended_min_max".